### ISSUE TITLE
[BUG] [Linter] Start of template tag is not recognized inside HTML tag

### ISSUE THREAD

**Issue by dyve** [Riverside-Healthcare/djLint#169]
- [ ] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [ ] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [ ] I've read the [docs](https://djlint.com)

## System Info
 - OS: macOS 12.1
 - Python Version: 3.8.12
 - djLint Version: 0.7.0
 - template language: Django

## How To Reproduce

Make a `temp.html` with this content:
```
<li{% if is_active %} class="active" {% endif %}>
    some content
</li>
```

And lint with this result:

```
❯ djlint temp.html

Linting 1/1 files ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 00:00


temp.html
───────────────────────────────────────────────────────────────────────────────
H025 1:0 Tag seems to be an orphan. <li{% if is_active %
H025 3:0 Tag seems to be an orphan. </li>

Linted 1 file, found 2 errors.
```

Now add a space after `li`

```
<li {% if is_active %} class="active" {% endif %}>
    some content
</li>
```

And lint again

```
❯ djlint temp.html

Linting 1/1 files ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 00:00


Linted 1 file, found 0 errors.
```

Either this is a bug (formatter should handle it) or a feature (formatter should report it, but not as an orphan). Because the Django `if` tag could even modify the tag, think of crazy stuff like

```jinja2
<li{% if foo %}nk rel="stylesheet" ... {% endif %}>{% if not foo %}no, we're a LI tag!</li>{% endif %}

<i{% if foo %}nput name="bar"{% endif %}>{% if not foo %}sorry, not allowed to input</i>{% endif %}
```

This might be something for the linter to catch ("XXX: Template language attempts to modify HTML tag").

Right now, it's not very predictable what this will do. See also #167.

## Configuration

```
[tool.djlint]
# Reference: https://djlint.com/djlint/rules.html
profile = "django"
ignore = "H006,H008,H013,H014,H017,H019,H021,H023,T001,T002,T003"
blank_line_after_tag = "load,extends,endblock,has_perm"
max_line_length = 120
```
